components/Footer.vue

```diff
@@ -1,9 +1,56 @@
 <template>
 	<footer>
-		<ContactInfo />
-		<SocialMedia />
-		<Funders />
-		<p>
+		<div class="footer-content">
+			<div class="nav-home-about">
```

Comment:
We should consider making these or perhaps all of the navigation here into a list, that way a screen reader user would know how many items are there. We should also consider wrapping them all in a `<nav>` element so that the user knows they are for navigation. Maybe check with @lliskovoi about her thoughts.

Reply:
I agree. It should be a list with secondary items as nested lists, and should be wrapped in a <nav> with aria-label="footer" or something along those lines.